armotypes/portaltypes.go

```diff
@@ -63,6 +63,7 @@ const (
 	AttributeApiVersion    = "apiVersion"
 	AttributeWorkloadHash  = "workloadHash"
 	AttributeIsIncomplete  = "isIncomplete"
+	AttributeSensor        = "clusterName"
```

Comment:
I think that the value should also be "sensor" and not "clusterName"

Reply:
value name is misleading, we have already attribute "cluster"
if attribute is hostSensor pls call it "hostSensor" if it's something else pls